README.md

```diff
@@ -91,6 +91,31 @@ FileInfo programFile = FileLocator.GetFileStartingAtSolutionDirectory("SourcePro
 
 [And more!](https://scadaminds.github.io/swiss-army-knife/md_Documentation_Files.html)
 
+
+## SwissArmyKnife.Compression
+Methods to compress and decompress data using gzip:
+```cs
+// Will return the compressed byte array
+var myCompressedByteArray = Gzip.Compress(myByteArray);
+
+// Will return the compressed string as byte array, using ASCII for encoding
+var myCompressedStringASCII = Gzip.Compress(myString, Encoding.ASCII);
+
+// Will return the compressed string as byte array, using UTF8 for encoding
```

Comment:
Instead of using `var` I think I would explicitly write the types out here, that'll be easier to read.